css/main.css

```diff
@@ -0,0 +1,964 @@
+}
+
+.l-client-item {
+  max-width: 85%;
+  order: 2;
+  text-align: center;
+  margin: auto;
+}
+
+.l-footer {
+  margin-bottom: 5%;
+  margin-top: 2%;
+}
+
+/*=== STYLES ===*/
+body {
+  font-family: 'Helvetica', 'Arial', Sans-serif;
+  font-weight: 400;
+}
+
+nav {
+  font-family: 'Open Sans', 'Helvetica', Sans-serif;
+  font-weight: 600;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+td {
+  background-color: #ffffff;
+  color: #2e2e2e;
+  font-size: 1.5em;
+}
+
+article {
+  padding: 2em 1em;
+}
+
+th, td {
+  padding: .25em 1em;
+  margin: .8em auto;
+}
+
+em {
+  font-style: none;
+}
+
+h2 {
+  width: 100%;
+  display: inline-block;
+}
+
+h2:before, h2:after {
+  content: " ";
+  display: inline-block;
+  border-top: solid 1px white;
+  width: 18%;
+  height: 1px;
+  z-index: 1;
+  position: relative;
+  margin: .25em;
+  white-space: nowrap;
+}
+
+.b-white h2:before, .b-white h2:after {
+  border-top: solid 1px black;
+}
+
+.h3-lines {
+  width: 100%;
+  display: inline-block;
+}
+
+.h3-lines:before, .h3-lines:after {
+  content: " ";
+  display: inline-block;
+  border-top: solid 1px white;
+  width: 18%;
+  height: 1px;
+  z-index: 1;
+  position: relative;
+  margin: .325em;
+  white-space: nowrap;
+}
+
+.b-white .h3-lines:before, .b-white .h3-lines:after {
+  border-top: solid 1px black;
+}
+
+.s-contact-info a, .s-contact-info a:hover, .s-contact-info a:visited {
+  text-decoration: none;
+  color: white;
+}
+
+.nav-link {
+  padding-top: 10%;
+}
+
+.nav-link a {
+  text-decoration: none;
+  color: white;
+  font-size: 1.2em;
+  padding-top: 2%;
+}
+
+.nav-link a:active {
+  color: #fd6632;
+}
+
+.hero-text-padding {
+  padding: 11em 5% 15em 5%;
+  margin-bottom: -5%;
+}
+
+/*=== PORTFOLIO HOVER ===*/
+.s-portfolio-image {
+  background-color: black;
+  position: relative;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+}
+
+.s-portfolio-image img {
+  display: block;
+}
+
+.portfolio-overlay-filter {
+  background-color: #000;
+  opacity: 0;
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  padding-top: 30%;
+  transition: opacity 0.5s ease-in 0s;
+}
+
+.s-portfolio-image:hover .portfolio-overlay-filter{
+  opacity: .6;
+}
+
+.portfolio-overlay-button {
+  height: 4em;
+  width: 2em;
+  text-align: center;
+  position: absolute;
+  left: 50%;
+  margin-left: -1.5em;
+}
+
+.portfolio-overlay-button a {
+  height: 58px;
+  width: 58px;
+  padding-left: 3px;
+  padding-right: 4px;
+  border-radius: 50%;
+  background-color: #fd6632;
+  text-align: center;
+  color: #ffffff;
+  color: white;
+  font-size: 3em;
+  line-height: 50px;
+  text-decoration: none;
+  display: block;
+}
+
+.portfolio-overlay-button a:hover {
+  background-color: #0b9ce0;
+}
+
+.first-level-hover-wrapper .portfolio-overlay-button {
+  transition: all 0.5s ease-in 0.5s;
+  top: -60px;
+}
+
+.first-level-hover-wrapper .item-wrapper:hover .portfolio-overlay-button {
+  top: 40%;
+}
+
+.b-orange,
+.b-orange-first {
```

Comment:
Is `.b-orange-first` being used? I don't see it in the document anywhere. If not, this should be updated.